packages/app-desktop/gui/NotePropertiesDialog.tsx

```diff
@@ -334,6 +336,8 @@ class NotePropertiesDialog extends React.Component<Props, State> {
 						}}
 						onKeyDown={event => onKeyDown(event)}
 						style={styles.input}
+						id={key}
+						name={key}
```

Comment:
Thank you for working on this!

Why is the `name={}` prop included here? (I think only `id` is necessary).